internal/services/notification/db/objects/objects.go

```diff
@@ -0,0 +1,40 @@
+package objects
+
+import (
+	"agola.io/agola/internal/sqlg"
+)
+
+const (
+	Version = uint(1)
+)
+
+const TypesImport = "agola.io/agola/services/notification/types"
+
+var AdditionalImports = []string{"time"}
+
+var ObjectsInfo = []sqlg.ObjectInfo{
+	{
+		Name: "RunWebhook", Table: "runwebhook",
+		Fields: []sqlg.ObjectField{
+			{Name: "Payload", Type: "string"},
```

Comment:
string type is wrong. It should be a byte array (blob).

To help you I added sqlg support for byte arrays and also testutil/db support (PR #419 #420)

Reply:
Thanks @sgotti I wait your PRs ar done(closed) to merge them to my pr

Reply:
You should first test them if they correctly work or something is missing.

I locally modified #413. It also requires fixing tests fixtures since they require a base64 input for Payload (now of type []byte).

Reply:
@sgotti I tested with your changes it works.

Reply:
Good. So please push the changes with the cherry picks of the other PR commits.